topdown/strings_test.go

```diff
@@ -0,0 +1,39 @@
+package topdown
+
+import (
+	"context"
+	"testing"
+
+	"github.com/open-policy-agent/opa/ast"
+)
+
+func TestSprintf(t *testing.T) {
```

Comment:
Perhaps rename this function to TestSprintfBigInt and also add a comment why this isn't possible to run from the test runner (yet)?